dashboard-app/src/app/pages/facility/views/facility.html

```diff
@@ -65,12 +65,16 @@
                         </div>
                     </div>
                     <div id="image" class="col-md-6" ba-panel ba-panel-title="Image">
-                        <div class="image">
+                        <div class="form-group" ng-class="{'has-error' : hotelFacility.facilityImage.$invalid && hotelFacility.$submitted, 'has-success' : hotelFacility.facilityImage.$valid && hotelFacility.$submitted}">
                             <img class="img-center" ng-src="{{imageSrc}}" ng-click="uploadPicture()"
-                                 height="272">
+                                 style="height: 258px">
```

Comment:
Don't use inline css. And still it is not same height, maybe it would be better to find different way instead of just setting height to fixed value.